CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.0-rc.6 (2018-11-20)
+
+ - Added support for `@skyux/list-builder-common@3.0.0-rc.1`. [#14](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/14)
+ - Updated peer dependencies to always take greater or equal versions. [#12](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/12)
```

Comment:
"Updated peer dependencies to support Angular versions greater than 4.3.6."
https://github.com/blackbaud/skyux-avatar/blob/master/CHANGELOG.md#300-rc3-2018-11-15

Reply:
Actually, that was a duplicate. It was released in rc.5. I'll remove it and update the same verbiage on the others, so we're consistent. Thanks!